platform_api_python_client/__init__.py

```diff
@@ -7,14 +7,14 @@
 
     No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
 
-    The version of the OpenAPI document: 0.1.0
+    The version of the OpenAPI document: 3.1.4
     Generated by OpenAPI Generator (https://openapi-generator.tech)
 
     Do not edit the class manually.
 """  # noqa: E501
 
 
-__version__ = "0.3.1"
+__version__ = "0.3.2"
```

Comment:
shouldn't this be 3.1.4 as well?

Reply:
This is the package version for the client. Not the Platform release version.